pkg/kubernetes/kubernetes.go

```diff
@@ -152,13 +153,27 @@ func (h *Client) WorkerIPs(ctx context.Context) (addrs []string, err error) {
 	addrs = []string{}
 
 	for _, node := range resp.Items {
-		if _, ok := node.Labels[constants.LabelNodeRoleMaster]; ok {
+		_, labelMaster := node.Labels[constants.LabelNodeRoleMaster]
+		_, labelControlPlane := node.Labels[constants.LabelNodeRoleMaster]
```

Comment:
This looks suspicious. Won't labelMaster and labelControlPlane be always the same?
Was the idea to use some other constant?

Reply:
yep :facepalm:, need different label there, thanks!